libsol/include/sol/transaction_summary.h

```diff
@@ -20,6 +20,8 @@
 // If all _Required_ `SummaryItem`s have not been set, finalization will fail.
 
 #define NUM_GENERAL_ITEMS 11
+#define DEFAULT_COMPUTE_UNIT_LIMIT 200000
+#define COMPUTE_UNIT_PRICE_DIVIDER 1000000
```

Comment:
Are those used anywhere?

Reply:
:thinking: good catch, It must been left from initial PR (It changed few times). I've removed it